docs/source/etl/variant-normalization.rst

```diff
@@ -57,17 +63,27 @@ The ``normalize_variants`` transformer has the following options:
+   * - ``reference_genome_path``
      - string
-     - Path to the reference genome ``.fasta`` or ``.fa`` file (required for normalization)
-
-       **Note**: ``.fai`` and ``.dict`` files with the same name must be present in the same folder.
-
+     - Path to the reference genome ``.fasta`` or ``.fa`` file. This file must be accompanied with a .fai index file in the same folder.
+   * - ``replace_columns``
+     - boolean
+     - |
+       | ``False``: The transformer does not touch the original ``start``, ``end``, ``referenceAllele`` and ``alternateAlleles`` columns. Instead, a StructType column called ``normalizationResult`` is added to the DataFrame which contains the normalized ``start``, ``end``, ``referenceAllele``, and ``alternateAlleles`` columns as well as the ``normalizationStatus`` StructType as the fifth field, which contains the following subfields:
+       |        ``changed``: Indicates whether the variant data was changed as a result of normalization or not.
+       |        ``errorMessage``: An error message in case the attempt at normalizing the row hit an error. In this case, the ``changed`` field will be set to ``false``. If no errors occur this field will be ``null``. In case of error, the first four fields in ``normalizationResult`` will be ``null``.
+       |
+       | ``True`` (default): The original ``start``, ``end``, ``referenceAllele``, and ``alternateAlleles`` columns are replaced with the normalized values in case they have changed. Otherwise (in case of no change or an error), the original ``start``, ``end``, ``referenceAllele``, and ``alternateAlleles`` are not touched. A StructType ``normalizationStatus`` column is added to the DataFrame with the same subfields explained above.
    * - ``mode`` (deprecated)
      - string
-     - | ``normalize``: Only normalizes the variants (if user does not pass the option, ``normalize`` is assumed as default)
+     - |
+       | ``normalize``: Only normalizes the variants (if user does not pass the option, ``normalize`` is assumed as default)
        | ``split_and_normalize``: Split multiallelic variants to biallelic variants and then normalize the variants. This usage is deprecated. Instead, use :ref:`split_multiallelics transformer<split_multiallelics>` followed by normalize_variants transformer.
        | ``split``: Only split the multiallelic variants to biallelic without normalizing. This usage is deprecated. Instead, use :ref:`split_multiallelics transformer<split_multiallelics>`.
 
-.. notebook:: .. etl/normalizevariants-transformer.html
+``normalize_variant`` Function
+==============================
+The normalizer can also be used as a SQL expression function applied on proper DataFrame columns to get normalization results as a new column. See the discreption of this function at :ref:`Glow PySpark Functions<pyspark_functions>` for details on how to use it in the Python API. The function can be similarly used in Scala API or as a SQL expression.
```

Comment:
Replace this section with: The normalizer can also be used as a SQL expression function. See :ref:`Glow PySpark Functions<pyspark_functions>` for details on how to use it in the Python API.

Reply:
We haven't explained how to use Scala API yet, so we should put off mentioning it for now. We can, however, create a code block demonstrating how to use the SQL expression.

Reply:
Done